wrapper/src/main/resources/messages.properties

```diff
@@ -120,7 +120,7 @@ DefaultConnectionPlugin.unknownRoleRequested=A HostSpec with a role of HostRole.
 # Driver
 Driver.nullUrl=Url is null.
 Driver.alreadyRegistered=Driver is already registered. It can only be registered once.
-Driver.missingDriver=Can''t find a suitable driver for ''{0}''.
+Driver.missingDriver=Can''t find the target driver for ''{0}''. Please ensure the target driver is in the classpath. Here is the list of registered drivers in the classpath: {1}
```

Comment:
Please ensure the target driver is in the classpath.
->
Please ensure the target driver is in the classpath and is registered.
?